### ISSUE TITLE
Change deployment to catalina ROOT 

### ISSUE THREAD

**Issue by pieterlukasse** [thehyve/cbioportal-docker#42]
Change deployment to catalina ROOT so that the final URL reads https://servername.com instead of https://servername.com/cbioportal.

**Comment by pieterlukasse** [comment]
In the Dockerfile it is as easy as (before the maven step)
```
RUN rm -rf $CATALINA_HOME/webapps/ROOT
```
and change 
```
	&& unzip portal/target/cbioportal-*.war -d $CATALINA_HOME/webapps/cbioportal \
```
to
```
	&& unzip portal/target/cbioportal-*.war -d $CATALINA_HOME/webapps/ROOT \
```
:warning: keep in mind related documentation should also be updated in cbioportal/cbioportal (e.g. like the SAML/Keycloak documentation containing URLs ending in /cbioportal).

**Comment by fedde-s** [comment]
I'll update the Docker deployment instructions once it gets documented in the upstream cBioPortal deployment manual.

**Comment by pieterlukasse** [comment]
Yes, ideally this can be done together, with 1 PR to cbioportal/cbioportal (for the documentation) and 1 PR here (for the Docker file).

**Comment by fedde-s** [comment]
@pieterlukasse, has your feature proposal been reported upstream? This docker setup will follow once it's a documented way to deploy cBioPortal; or maybe as soon as it's in progress there if it immediately greatly benefits our deployments, like we did with https://github.com/cBioPortal/cbioportal/issues/4832.

**Comment by pieterlukasse** [comment]
No, there is no upstream PR yet.

**Comment by pieterlukasse** [comment]
Is this file already solving this issue...and introducing a new one (namely a documentation issue)?
https://github.com/cBioPortal/cbioportal/blob/master/docker/web/Dockerfile
CC @inodb
